### ISSUE TITLE
Known issues, problems, limitations w/ IKEA KADRILJ smart blinds?

### ISSUE THREAD

**Issue by mtressl** [dresden-elektronik/deconz-rest-plugin#3599]
## Describe the question or issue you are having

Okay, so first things first: the blinds were working before, either through deConz, Homebridge, or HomeKit.
But around 10 days ago they suddenly stopped working. So I did what I did before, tried to reconnect the blinds to deConz, reset the blinds, re-pair the blinds, delete them through deConz, Phoscon, even through the API directly.

Result is, the blinds connect, I can see them in deConz and Phoscon, can read stuff - but they do.not.move anymore.

Strangest thing is, that the battery indicator in deConz shows a low battery even though the battery is fully charged. I even got a new one from IKEA - same problem, see screenshot below.

I am running `deconz:latest` (2.05.86). Already downgraded several times to `deconz:stable` (2.05.84).
My RaspBee II is running the latest firmware (26660700) as well.

Ok, my question:

are there known issues that I did not find here in the reports that cause the behaviour I have?
I am really annoyed b/c of this behaviour, and before I throw the blinds away I wanted to check w/ you guys.

## Screenshots

![image](https://user-images.githubusercontent.com/1140154/98362426-7e544080-202d-11eb-8eca-226f9c7004f4.png)
![image](https://user-images.githubusercontent.com/1140154/98362439-83b18b00-202d-11eb-9b59-58d942eaff64.png)
![image](https://user-images.githubusercontent.com/1140154/98362449-87451200-202d-11eb-8a12-516856692b28.png)
![image](https://user-images.githubusercontent.com/1140154/98362690-fcb0e280-202d-11eb-8f9c-77b5809f18c8.png)

## Environment

 - Host system: Raspberry Pi
 - Running method: Marthoc Docker container
 - Firmware version: 26660700
 - deCONZ version: 2.05.86 / 2.05.84
 - Device: RaspBee II
 - Is there any other USB or serial devices connected to the host system? no

**Comment by wvuyk** [comment]
Did you try the battery trick? https://github.com/dresden-elektronik/deconz-rest-plugin/issues/1121#issuecomment-702055261

**Comment by mtressl** [comment]
> Did you try the battery trick? [#1121 (comment)](https://github.com/dresden-elektronik/deconz-rest-plugin/issues/1121#issuecomment-702055261)

Yes, you mean out-wait-in-again? I did... and that's why I got a new battery from IKEA, b/c I wanted to make sure it's not a defect battery pack.

**Comment by wvuyk** [comment]
Not 100%. When you take it out, start a scan for new devices. Then put it back in. That at least is how I did it with success?

**Comment by mtressl** [comment]
Just to completely understand your process:
- blinds are still added to the system (but faulty battery display and not moving /reacting)
- I take the battery out
- start new device discovery from Phoscon (or deConz?)
- re-insert the battery (but no reset on the blinds)

Correct?

**Comment by wvuyk** [comment]
Yes, that is has been working for me in the past. Do not remove the device from deconz, the scan will refresh the device, that is all.
You could also try to read the basic cluster during the scan. For some devices this helps the refresh as well.

**Comment by mtressl** [comment]
Unfortunately none of the proposed solutions do the trick @wvuyk - I cannot get the battery indicator to refresh and show a fully charged state (which the battery is). Any other hints and tipps how to fully reset the blinds, or make deConz completely starting from scratch with the blinds? Speaking of starting from scratch, removing the blinds through deConz, Phoscon, and through the API should be all to consider when doing this, correct?

**Comment by wvuyk** [comment]
I think the only trustworthy removal is through Phoscon, but am not sure on that as I keep seeing ghost devices returning back in my installation in the deConz map here. Whatever I try, they keep comming back But that could also be remnants of old beta stuff running here...

I found an old remark here in my notes that for some blinds you would have to manually set up the attribute reporting, is this something you have tried?

Other then that, I am out of suggestions....

**Comment by mtressl** [comment]
> manually set up the attribute reporting

No I have not tried that. Can you point me in any direction where I can read about it?

**Comment by wvuyk** [comment]
I am sure it wil be somewhere here too, but here is a link to another post for a device: https://github.com/dresden-elektronik/deconz-rest-plugin/issues/3226#issuecomment-685960083, I do not know if this was posted in a wiki as well.
Also[ here on my own forum](https://forums.homeseer.com/forum/hs4-products/hs4-plugins/lighting-primary-technology-plug-ins-aa/jowihue-w-vuyk-aa/1398707-using-a-ikea-smart-blind-fyrtur-or-kadrilj)

**Comment by mtressl** [comment]
Manual attribute reporting did not help. I also managed to re-flash the firmware of the blinds, but no success in controlling.
What I start seeing is the following line in my Homebridge logs:
`[11/10/2020, 8:15:48 PM] [Philips Hue & deCONZ (ZigBee)] Window covering: set homekit last boot from Tue Nov 10 2020 20:12:32 to Tue Nov 10 2020 20:15:48`

Does that point towards an endless reboot loop on the blinds? Might they actually be broken?

**Comment by mtressl** [comment]
@ebaauw 
Sorry for tagging you out of context, please see my problems above.
As it seems the homebridge-hue plugin gives me the following log entry, not sure if this might be related to my general issue: `Window covering: set homekit last boot from Wed Nov 11 2020 17:20:33 to Wed Nov 11 2020 17:23:25`

**Comment by Dodoooh** [comment]
is there a newer / better way. Unfortunately I still have problems with adding the roller shutter.
I have tried it as described here: https://github.com/dresden-elektronik/deconz-rest-plugin/issues/1121#issuecomment-524617659

in home assistant i unfortunately only get the battery status displayed, but not the shutter itself. but i can open / close it in the deconz GUI

**Comment by Smanar** [comment]
I think you just don't find it in HA.
If you can find it in phoscon, it mean it exist in APÏ, so need to be visible in HA too.

You have no fonction to resynchronise ?

**Comment by Dodoooh** [comment]
> I think you just don't find it in HA.
> If you can find it in phoscon, it mean it exist in APÏ, so need to be visible in HA too.
> 
> You have no fonction to resynchronise ?

but it should be displayed in home assistant, i mean how else could i control the shutters? 

I also tried with the NodeRed - Deconz plugin which has direct api access, there it also does not find the shutter :/

**Comment by Smanar** [comment]
How many device have you ?
Have you take a look in the light device list ? (with Node red if it show the complete list ?)

**Comment by Dodoooh** [comment]
i have 70 nodes

I can't find anything in node red except the battery. i was able to integrate it half a year ago, and there i could find it in node red as well as in home assistant....

i switched to a new infrastructure and wanted to do everything from scratch. to eliminate first errors for all time. was apparently not the best idea....
<img width="526" alt="Bildschirmfoto 2020-12-18 um 20 54 59" src="https://user-images.githubusercontent.com/5958446/102655872-4cc5ad80-4173-11eb-96f1-fd23cd717d82.png">
<img width="1359" alt="Bildschirmfoto 2020-12-18 um 20 52 39" src="https://user-images.githubusercontent.com/5958446/102655668-ffe1d700-4172-11eb-8c39-9a8742757e27.png">

**Comment by Smanar** [comment]
You are not loking a good place, in the light section there is no battery device.

You have an API key that you are using in third app, just go on 

`http://IP:PORT/api/KEY/lights`

IP znd PORT are the same you are using for phoscon.

**Comment by SwoopX** [comment]
The blinds do not seem to be paired correctly, you can see that on the "unusual" names.

**Comment by Dodoooh** [comment]
@Smanar. no nothing like that can be found here. Only my lamps and other such devices.. 

@SwoopX how to pair them "even more" ? :) I can control them via Web GUI

**Comment by SwoopX** [comment]
> I can control them via Web GUI

Any device can be controlled from the GUI once they successfully received the network settings and are allowed to join the network. But this is just the first half of the join processes. The second does the lights and sensors creation, which can be seen by the device NWK address as its name and confirmed by you not being able to find any REST resource.

Try entering light search and then open/close the blind. If that's unsuccessful, remove the batteries for 10 secs and try again.

**Comment by Dodoooh** [comment]
thank you @SwoopX but it doesn't look like it solved the problem. :/ i've done your process several times now and i'm still in the same situation.

**Comment by SwoopX** [comment]
Hm, if a sensor search also doesn't help, I'm out I'm afraid. Don't have any :/

**Comment by Smanar** [comment]
But if it work on phoscon, it mean the API entry is somewhere, no ?

**Comment by Dodoooh** [comment]
so I do not know what it was, possibly an early Christmas miracle ;) (I sat surely half a day with three shutters in a small storage room to be close enough to the deCONZ stick) 

After hours of various factory resets and re-adds I came across a Github post (link will come as soon as found): 

Once shortly press both buttons. 
Then press both buttons for about 30 seconds +
After that normal factory reset.

So it was found immediately in the Phoscon WebGUI and added I did not have to bind anything via VNC. 

Thank you again very much for your support! @SwoopX @Smanar
